tests/test_benchmark.py

```diff
@@ -351,7 +351,7 @@ def setUpClass(self):
     assert(os.path.exists(CLOSURE_COMPILER))
 
     Building.COMPILER = CLANG
-    self.emcc_args += [OPTIMIZATIONS]
+    self.emcc_args = [OPTIMIZATIONS]
```

Comment:
Hmm.. I'm not sure this will work.. since the setUp method which runs for each tests will reset it, no?  Is it working for you?

I think this should be part of setUp not setUpClass.  While you are here you should probably rename `self` here to `cls`?

Reply:
Thanks - it does run the benchmarks, but yeah, I think it's not actually using the flag. Which means it may not be optimizing properly. I'll look into it more when I get a chance.